src/scss/src/foundation/_variables.scss

```diff
@@ -297,3 +297,20 @@ $spacing: (
 );
 
 /*=====  End of BREAKPOINTS  ======*/
+
+/*=============================================
+=                  SHADOWS                   =
+=============================================*/
+
+$mainShadow:
+  0px 5px 6px -3px rgba(144, 164, 174, 0.2),
+  0px 9px 12px 1px rgba(144, 164, 174, 0.14),
+  0px 3px 16px 2px rgba(144, 164, 174, 0.12);
+
+$commonShadow: 0px 3px 16px 2px rgba(144, 164, 174, 0.12);
+
+$commonHoverShadow: 0px 3px 16px 2px rgba(144, 164, 174, 0.56);
+
+$smallHoverShadow: 0 3px 1px rgba(144, 164, 174, 0.56);
```

Comment:
Also use cebab case for naming shadows variables

Reply:
makes sense, solved